eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/OperationMethodProviders/ResourceOperationMethodProvider.cs

```diff
@@ -25,11 +25,14 @@ namespace Azure.Generator.Management.Providers.OperationMethodProviders
     /// </summary>
     internal class ResourceOperationMethodProvider
```

Comment:
do we handle anything special for resource collection operation methods here? This leads to the question in https://github.com/Azure/azure-sdk-for-net/pull/51169/files#r2199449190